test/test_reductions.py

```diff
@@ -3382,6 +3383,23 @@ def to_numpy(input):
 
             self.assertEqual(actual, expected, msg, exact_dtype=exact_dtype)
 
+    @onlyCUDA
+    @largeTensorTest("8GB")
+    @dtypes(torch.half, torch.chalf, torch.bfloat16)
```

Comment:
Well, this test fails on large GPUs with expected "mean_cuda is not implemented for 'ComplexHalf'"

Reply:
I see that skip has been added. Thanks for checking and adding skip.